pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/JdbiStudyLanguage.java

```diff
@@ -0,0 +1,13 @@
+package org.broadinstitute.ddp.db.dao;
+
+import org.jdbi.v3.sqlobject.customizer.Bind;
+import org.jdbi.v3.sqlobject.statement.SqlUpdate;
+
+public interface JdbiStudyLanguage {
+    @SqlUpdate(
```

Comment:
I'll recommend changing return type to `long` and adding `@GetGeneratedKeys` since the table now have an auto-increment PK.